deployments/puppet/metadata.json

```diff
@@ -9,6 +9,14 @@
     {
       "name": "puppetlabs/apt",
       "version_requirement": ">= 4.1.0 <= 7.0.0"
+    },
+    {
+      "name": "puppet/archive",
+      "version_requirement": ">= 5.5.8 < 7.0.0"
```

Comment:
@asuresh4 are these versions correct? I only see 0.4.4 to 4.1.0 at https://forge.puppet.com/puppet/archive.

Reply:
Oh I put the puppet versions in there. Let me update that

Reply:
corrected